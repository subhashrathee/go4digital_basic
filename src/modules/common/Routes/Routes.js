import React from 'react'
import { Route } from 'react-router-dom'

import Home from '../../pages/Home/Home'
import Services from '../../pages/Services/Services'
import About from '../../pages/About/About'
import Blog from '../../pages/Blog/Blog'
import Contact from '../../pages/Contact/Contact'

/**
* The main Route class. This sets up the routes for the header component.
*
* Props:      None
* 
* State vars: None
*/
class Routes extends React.Component {
  render() {
    return (
        <div>   
          <Route exact path="/" component={Home}/>
          <Route path="/services/" component={Services}/>
          <Route path="/about_us/" component={About}/>
          <Route path="/blog/" component={Blog}/>
          <Route path="/contact_us/" component={Contact}/>
        </div>
    )
  }
}

export default Routes;
