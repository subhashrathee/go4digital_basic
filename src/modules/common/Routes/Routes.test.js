import React from 'react';
import Routes from './Routes';
import renderer from 'react-test-renderer';
import { BrowserRouter, Link } from 'react-router-dom'

test('Route component', () => {
  const component = renderer.create(
    <BrowserRouter>
    <Routes />
    </BrowserRouter>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
