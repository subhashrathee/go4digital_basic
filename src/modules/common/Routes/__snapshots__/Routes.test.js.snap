// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route component 1`] = `
<div>
  <div>
    <div
      className="hero-image-home"
    />
    <div
      className="main"
    >
      <h1>
        go4digital: digital journey together
      </h1>
      <p>
        Congratulations on deciding to start your business's digital journey. Believe me, you are not alone. Travel this digital journey together with us as we just embarked on it.
      </p>
      <p>
        First step in the journey is to get a website, we have got our domain name, website hosting already sorted, now working through to the look and feel and digital content of our website. We can also help you out to reach to the same stage before we can work together to establish our businesses in the digital world
      </p>
      <p>
        Next step in the journey is to share your digital website address with your existing clients so that they can reach you as per their convenience and need. Get your website up and running and start sharing your business's web address with your existing and potential customers. Feel free to reach out and learn more on this.
      </p>
      <p>
        After having a web address and catering for your existing loyal customers' convenience, now is the time to try and get some more business. Thumb rule is, your digital + social media reputation leads to new business opportunities. Wondering how can you do that? Embark on this journey and we will educate you about this and partner with you on establishing your business brand on digital landscape.
      </p>
      <p>
        Run digital campaigns to reach out thousands of customers looking for services you offer. Want to know how? Reach out and ask for campaign packages and choose right plan as per your budget and business needs
      </p>
    </div>
    <footer>
      <p>
        Â© 
        2018
         go4digital. All rights reserved.
      </p>
    </footer>
  </div>
</div>
`;
